content/docs/09-registries-and-packs/1-adding-a-custom-registry.md

```diff
@@ -36,10 +36,61 @@ Pack contents are periodically synchronized with Palette.
 Please ensure that the ports 443 and 80 are exclusively allocated to the registry server and are not in use by other processes.
 </WarningBox>
 
-# Deploying a Pack Registry Server
-
-Palette provides a Docker image for the pack registry server. The following steps need to be performed to deploy the pack registry server using this docker image:
-
+# Deploying a Pack Registry Server with letsencrypt certificates
+
+The following section demonstrates how you can deploy a registry server secured with a TLS certificate issued by [Let's Encrypt](https://letsencrypt.org/).
+For more advanced configuration and deployment with self-signed certificates see [advanced configuration.](/registries-and-packs/advanced-configuration)
```

Comment:
```suggestion

Check out the [advanced configuration](/registries-and-packs/adding-a-custom-registry#self-signedcertificates) section for more advanced configuration and deployment with self-signed certificates.
```
Added a newline and changed the order to improve the flow.

Reply:
Fixed.